rel-eng/tito.props

```diff
@@ -574,17 +574,18 @@ whitelist = katello
   rubygem-hammer_cli_foreman_virt_who_configure
   rubygem-hammer_cli_katello
   rubygem-katello
+  rubygem-pulp_2to3_migration_client
   rubygem-pulp_ansible_client
-  rubygem-pulpcore_client
   rubygem-pulp_container_client
+  rubygem-pulpcore_client
```

Comment:
This sorting looks odd

Reply:
It's because `LC_COLLATE=en_GB` in add_gem_package.sh

LC_COLLAGE=C seems to fix it